src/app/components/About/About.jsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 
 const Sidebar = () => (
-  <div className="about">
-    <h3>Additional Info</h3>
-    <p>Many of these titles are available in formats for <a href="">patrons
-    with print disabilities.</a></p>
-    <p><a href="">About Best Books</a></p>
-    <ul>
-      <li>Facebook icon</li>
-      <li>Twitter icon</li>
-    </ul>
+  <div className="about nypl-row">
+    <div className="inner">
+      <h3>Additional Information</h3>
+      <p>Many of these titles are available in formats
+        for <a href="https://www.nypl.org/accessibility/print-disabilities">patrons
+        with print disabilities.</a></p>
+      <p><a href="https://www.nypl.org/books-music-dvds/recommendations/about/annual-lists">About
```

Comment:
you can probably move the <a> to a new line for easier reading. Can these URL's be in the appConfig, in the event of a change, we know all urls are in one central place?

Reply:
Cool, updated.